frontend/appflowy_flutter/lib/plugins/ai_chat/presentation/chat_input/mobile_ai_prompt_input.dart

```diff
@@ -265,6 +269,11 @@ class _MobileAIPromptInputState extends State<MobileAIPromptInput> {
               AIType.appflowyAI => LocaleKeys.chat_inputMessageHint.tr(),
               AIType.localAI => LocaleKeys.chat_inputLocalAIMessageHint.tr()
             },
+            hintStyle: Theme.of(context).textTheme.bodyMedium?.copyWith(
```

Comment:
You can move these colors to an extension file to make it easier to refactor in the future if we have the shared color palette.

Reply:
Do you think I should move it just to the scope of the chat component or even wider? I don't know how stable the colored palette is at this moment

Reply:
Moving to the chat component is enough for now because the colors used here are hardcoded and same, just to ensure that these widgets are using the same color.

Reply:
OK!